ci/kokoro/README.md

```diff
@@ -33,95 +32,3 @@ Somewhat unique to Kokoro one must define a separate *INTEGRATION* vs.
 *PRESUBMIT* thus the duplication of configuration files for essentially the
 same settings.
 
```

Comment:
Trailing blank line?

Reply:
Removed. We should have a `checkers` phase that checks for this. I'll think if I can come up with something. If you know of an easy way to check this, let me know.